deploy/common/k8sbased.sh

```diff
@@ -473,6 +473,7 @@ function launch_central {
           if [[ "${ROX_SCANNER_V4:-}" != "false" ]]; then
             echo "Deploying ScannerV4..."
```

Comment:
While we are at it, could you move this `echo` one line below the following `if...; then`?